### PR TITLE
feat(build): enable static link to SPDK libs

### PR DESCRIPTION
**Pull request by dsavitskiy**

### CODE REVIEW

File: nix/pkgs/io-engine/cargo-package.nix [hunk 1]
```diff
@@ -86,23 +86,24 @@ let
...
+#      for bin in "$out/bin/"*; do
+#        new_rpath=$(patchelf --print-rpath "$bin" | sed -r 's/\/build(\/[^:]*)+/'"$ms_lib_path"'/')
+#        patchelf \
+#            --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
+#            --set-rpath "$new_rpath" \
+#            "$bin"
+#      done
+#    '';
+#    outputs = [ "out" "lib" ];
+    outputs = [ "out" ];
```
Comment (tiagolobocastro) [inline nix/pkgs/io-engine/cargo-package.nix]:
```suggestion
```

**Review by tiagolobocastro — APPROVED** [review]

**Review by hrudaya21 — APPROVED** [review]

File: nix/pkgs/io-engine/cargo-package.nix (hunk 1 above)
Comment (dsavitskiy) [reply nix/pkgs/io-engine/cargo-package.nix]: ok

**Comment by dsavitskiy** [comment]
bors merge

**Comment by dsavitskiy** [comment]
bors merge

**Pull request merged**